.github/workflows/development-environment.yml

```diff
@@ -26,22 +31,18 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Set environment variables & others
-        if: ${{ runner.os == 'macOS' }}
         run: |
           echo "VOLTA_HOME=$HOME/.volta" >> $GITHUB_ENV
           echo "PATH=$HOME/.volta/bin:$PATH" >> $GITHUB_ENV
           touch "$HOME/.zshrc"
 
       - name: Install prerequisites
-        if: ${{ runner.os == 'macOS' }}
-        # Depending on the host watchman will not install (read: intermittent issue),
-        # thus, we mark this as continue-on-error
-        continue-on-error: true
```

Comment:
I want to see if this issue happens anymore now that we try two times to execute `brew bundle`.